NitroxServer/GameLogic/Bases/BaseManager.cs

```diff
@@ -5,8 +5,10 @@
 using System.Linq;
 using NitroxModel.DataStructures.Util;
 
+
 namespace NitroxServer.GameLogic.Bases
 {
+    [System.Runtime.InteropServices.ComVisible(true)]
```

Comment:
Let's remove this, we have better keyword for managing visibility inside a class. Besides the default value of this attribute is true (cf [ComVisibleAttribute MicrosoftDocs](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comvisibleattribute))

Reply:
Why is this comment marked resolved?

Reply:
Does it need to be removed or replaced, I did not add this, this was there when I got my hands on this code